python/tidy/servo_tidy_tests/test_tidy.py

```diff
@@ -13,6 +13,7 @@
 from servo_tidy import tidy
 
 base_path = 'servo_tidy_tests/' if os.path.exists('servo_tidy_tests/') else 'python/tidy/servo_tidy_tests/'
+COMPILE_FAIL_PATH = '../../../tests/compiletest/plugin/compile-fail/'
```

Comment:
We should be adding new tests to this directory since we'll be removing the existing compile-fail tests.